### DISCUSSION TITLE
laravel new xxxx,   but it does not match the constraint.

### DISCUSSION THREAD

**Discussion by huaiguoguo** [laravel/framework#47501] [Q&A]
### Laravel Version

10.2.3

### PHP Version

8.2.4

### Description

```cmd

F:\PhpstormProjects>composer create-project laravel/laravel sea_wave
Creating a "laravel/laravel" project at "./sea_wave"
Installing laravel/laravel (v10.2.2)
  - Downloading laravel/laravel (v10.2.2)
  - Installing laravel/laravel (v10.2.2): Extracting archive
Created project in F:\PhpstormProjects\sea_wave
> @php -r "file_exists('.env') || copy('.env.example', '.env');"
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laravel/framework[v10.10.0, ..., v10.12.0] require dragonmantank/cron-expression ^3.3.2 -> found dragonmantank/cron-expression[dev-master, v1.0.0, ..., v1.2.1, v2.0.0, ..., v2.3.1, v3.0.0, ..., v3.3.1] but it does not match the constraint.
    - Root composer.json requires laravel/framework ^10.10 -> satisfiable by laravel/framework[v10.10.0, v10.10.1, v10.11.0, v10.12.0].

```

### Steps To Reproduce

```cmd

F:\PhpstormProjects>composer create-project laravel/laravel sea_wave
Creating a "laravel/laravel" project at "./sea_wave"
Installing laravel/laravel (v10.2.2)
  - Downloading laravel/laravel (v10.2.2)
  - Installing laravel/laravel (v10.2.2): Extracting archive
Created project in F:\PhpstormProjects\sea_wave
> @php -r "file_exists('.env') || copy('.env.example', '.env');"
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laravel/framework[v10.10.0, ..., v10.12.0] require dragonmantank/cron-expression ^3.3.2 -> found dragonmantank/cron-expression[dev-master, v1.0.0, ..., v1.2.1, v2.0.0, ..., v2.3.1, v3.0.0, ..., v3.3.1] but it does not match the constraint.
    - Root composer.json requires laravel/framework ^10.10 -> satisfiable by laravel/framework[v10.10.0, v10.10.1, v10.11.0, v10.12.0].

```

**Comment by ghost** [comment]
Try a newer version of Laravel. You have version `10.2.3`, the latest is `10.13.5` and you need at least version `10.10`.

**Reply by huaiguoguo** [reply]
i am a  chininese developer, i have used mirroring,  this is an issue with mirroring and has nothing to do with "laravel".
thanks again.

**Comment by lifetin** [comment]
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires laravel/framework ^10.15.0 -> satisfiable by laravel/framework[v10.15.0].
    - laravel/framework v10.15.0 requires dragonmantank/cron-expression ^3.3.2 -> found dragonmantank/cron-expression[dev-master, v1.0.0, ..., v1.2.1, v2.0.0, ..., v2.3.1, v3.0.0, ..., v3.3.1] but it does not match the constraint.
  Problem 2
    - phpunit/phpunit[10.1.0, ..., 10.2.4] require phpunit/php-text-template ^3.0 -> found phpunit/php-text-template[dev-main, 3.0.x-dev (alias of dev-main)] but it does not match your minimum-stability.
    - Root composer.json requires phpunit/phpunit ^10.1 -> satisfiable by phpunit/phpunit[10.1.0, ..., 10.2.4].




我从10.10.0/10.2.3/10.13.5 都试了,不行,一直是失败
找不到   dragonmantank/cron-expression ^3.3.2
